src/p5.dance.js

```diff
@@ -1119,7 +1119,25 @@ module.exports = class DanceParty {
   }
 
   // Called when executing the AI block.
-  async ai(value) {
+  ai(params) {
+    console.log('handle AI:', params);
```

Comment:
Did you intend to leave this print statements here?

Reply:
Yes, I have found them useful for development, for now.  We can probably remove when we get closer to done.